src/routes/Ecosystem/Common/VaultField.styled.tsx

```diff
@@ -0,0 +1,61 @@
+import Box from '@mui/material/Box';
+import { styled } from '@mui/material/styles';
+import Typography from '@mui/material/Typography';
+
+import { Typography as CustomTypography } from '../../../components/atomic/Typography/Typography';
+
+export const VaultFieldBox = styled(Box)`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: ${({ theme }) => theme.spacing(4)};
+`;
+
+export const TitleBox = styled(Box)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0px;
+  gap: ${({ theme }) => theme.spacing(4)};
+`;
+
+export const TitleTypography = styled(Typography)`
+  font-family: 'PixelOperatorMono', monospace;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 130%;
+
+  letter-spacing: 0.02em;
+
+  color: #e5e1f9;
```

Comment:
Perhaps use a constant from colors for this. I think it is lavenderWeb

Reply:
Done. https://github.com/Voltz-Protocol/voltz-ui/commit/d6cedb705445c150c0de0460c9ff447f5ea867d3